src/main/java/org/javarosa/core/model/instance/TreeReference.java

```diff
@@ -483,14 +483,16 @@ public int hashCode() {
     }
 
     public String toString() {
-        return toString(true);
+        return toString(true, true);
     }
 
     public String toString(boolean includePredicates) {
```

Comment:
I don't think that still taking in parameters but overriding them is a good idea. Could you mark those two variants as deprecated instead?